stable/postgresql/values-production.yaml

```diff
@@ -180,6 +180,11 @@ service:
   ##
   # loadBalancerIP:
 
+  ## Load Balancer sources
+  ## https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+  # loadBalancerSourceRanges:
```

Comment:
Can you add a line only with ## between the comment and the parameter? Check other examples:
```yaml
  ## RabbitMQ Manager port
  ## ref: https://github.com/bitnami/bitnami-docker-rabbitmq#environment-variables
  ##
  managerPort: 15672
```

Reply:
ok added